charts/kong/FAQs.md

```diff
@@ -137,3 +137,29 @@ for setting a password:
 If you have already upgraded, the old password is lost. You will need to
 delete the Helm release and the Postgres PersistentVolumeClaim before
 re-installing with a non-random password.
+
+### Kong's admission webhook interrupts Kubernetes control plane operations
+
+Kubernetes control plane components rely on resources within the `kube-system` namespace.
+Modifying or rejecting requests in this namespace using an admission webhook may cause
+unintended disruptions to cluster operations, including failures in critical services like
+`kube-dns`. By default, Kong's admission webhook is configured with no namespace selector, 
```

Comment:
I'm wondering if replacing `Kong` with `KIC` wouldn't be better here since KIC is the actual owner of said webhook (and a controller for objects validated by it). WDYT?

```suggestion
`kube-dns`. By default, KIC's admission webhook is configured with no namespace selector, 
```